libafl_libfuzzer/runtime/Cargo.toml.template

```diff
@@ -4,16 +4,19 @@ version = "0.15.2"
 edition = "2024"
 publish = false
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [features]
-default = ["fork"]
+default = []
```

Comment:
I think fork as default is good for perf on non-windows, it's a not on win

Reply:
I have it set below that on non-windows to keep things explicit and because tui_monitor needs to be default on non-win anyway, but I can change it if wanted.